content/docs/archive/vmclarity/getting-started/first-tasks/_index.md

```diff
@@ -15,30 +15,30 @@ weight: 200
 
 1. Click on the **Scans** icon. In the Scans window, you can create a new scan configuration.
 
-    <img src="/img/vmclarity-ui-2.png" alt="VMClarity UI Scan" title="VMClarity UI Scan" />
+    <img src="/img/archive-vmclarity-ui-2.png" alt="VMClarity UI Scan" title="VMClarity UI Scan" />
```

Comment:
Hey, please rebase to the latest main, because these were already changed [here](https://github.com/openclarity/docs.openclarity.io/pull/50) 🙂

Reply:
Sorry I looked at the wrong file, everything is fine! 😊